opal/mca/pmix/base/pmix_base_select.c

```diff
@@ -29,6 +29,16 @@ int opal_pmix_base_select(void)
     opal_pmix_base_component_t *best_component = NULL;
     opal_pmix_base_module_t *best_module = NULL;
 
+    // Set the PMIx component_show_load_errors MCA param based on the
+    // value of OPAL's mca_base_component_show_load_errors (i.e., the
+    // bool variable behind Open MPI's mca_component_show_load_errors
+    // MCA param).
+    char *pmix_show_load_errors_env = NULL;
+    asprintf(&pmix_show_load_errors_env,
+             "PMIX_MCA_mca_component_show_load_errors=%d",
```

Comment:
Should that be `PMIX_MCA_mca_base_component_show_load_errors` I think the `base` is needed because without it there is a deprecated warning, right?

Reply:
Fixed.